index.bs

```diff
@@ -795,6 +743,562 @@ the following list:
 
 </div>
 
+
+
+# The {{WebSocketStream}} interface # {#the-websocketstream-interface}
+
+The Web IDL definition for the {{WebSocketStream}} class is given as follows:
+
+<xmp class="idl">
+dictionary WebSocketOpenInfo {
+  ReadableStream readable;
+  WritableStream writable;
+  DOMString extensions;
+  DOMString protocol;
+};
+
+dictionary WebSocketCloseInfo {
+  [Clamp] unsigned short closeCode;
```

Comment:
`[Clamp]` is not generally recommended; instead prefer `[EnforceRange]`.

Reply:
Done.